script.js

```diff
@@ -0,0 +1,92 @@
+const duration = document.querySelectorAll('.duration__minutes')
+const timer = document.querySelector('.control__element--time')
+const replay = document.querySelector('.control__element--replay')
+const play = document.querySelector('.control__element--play')
+const progressLine = document.querySelector('.control__circle')
+const video = document.querySelector('.video')
+const firstSwitcherButton = document.querySelector('.switcher-video__video--first')
+const secondSwitcherButton = document.querySelector('.switcher-video__video--second')
+const sound = document.querySelector('.sound')
+
+let isPlayed = false
+let isTimerActive = false
+let currentduration = 10
+let newTime = currentduration * 60000
+let interval
+
+duration.forEach((button) => {
+
+    button.addEventListener('click', () => {
+        currentduration = button.textContent.split(' ')[0]
+        newTime = currentduration * 60000
```

Comment:
What is 6000? Use variables to name such things. E.g, const MINUTES_IN_HOUR = 60.  It is quite normal. Besides, you use it some times (line 14, line 21 and later in the code).  DRY principle. One variable - many usages.